openhands/server/session/agent_session.py

```diff
@@ -178,6 +178,10 @@ async def stop_agent_loop_for_error(self):
         if self.controller is not None:
             await self.controller.set_agent_state_to(AgentState.ERROR)
 
+    def get_app_pprts(self):
```

Comment:
This name looks a bit weird. Something else like `get_runtime_info_prompts` might be better